[2022-11-30T07:02:15.986+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2022-11-30T07:02:16.007+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2022-11-30T07:02:16.007+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T07:02:16.007+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-30T07:02:16.007+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T07:02:16.032+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): world> on 2022-08-29 00:00:00+00:00
[2022-11-30T07:02:16.038+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'scheduled__2022-08-29T00:00:00+00:00', '--job-id', '496', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpo2qu4aob']
[2022-11-30T07:02:16.035+0000] {standard_task_runner.py:55} INFO - Started process 1963 to run task
[2022-11-30T07:02:16.039+0000] {standard_task_runner.py:83} INFO - Job 496: Subtask world
[2022-11-30T07:02:16.194+0000] {task_command.py:376} INFO - Running <TaskInstance: first_dag.world scheduled__2022-08-29T00:00:00+00:00 [running]> on host 733208eb192a
[2022-11-30T07:02:16.376+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-08-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-29T00:00:00+00:00
[2022-11-30T07:02:16.376+0000] {logging_mixin.py:137} INFO - Hello!
[2022-11-30T07:02:16.376+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-30T07:02:16.413+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20220829T000000, start_date=20221130T070215, end_date=20221130T070216
[2022-11-30T07:02:16.458+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-30T07:02:16.493+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
